src/components/Slider.vue

```diff
@@ -70,15 +76,35 @@ export default class Slider extends Vue {
   @Prop()
   data!: Array<number> | Array<string>;
 
-  displayValue: number | string | Array<number> | Array<string> = this.value;
-
   @Prop({ default: false })
   simpleTheme!: boolean;
 
+  private displayValue: number | string | Array<number> | Array<string> = this
```

Comment:
i don't think this should be a private variable if it's accessed by the template, right?

Reply:
I dont think it should matter, but im going to remove the private, because it will work either way.